tools/cjxl_ng_main.cc

```diff
@@ -65,26 +76,17 @@ DEFINE_bool(qprogressive_ac, false, "Use progressive mode for AC.");
 
 DEFINE_bool(modular_lossy_palette, false, "Use delta-palette.");
 
-DEFINE_bool(jpeg_transcode, false,  // TODO(tfish): Wire this up.
-            "Do lossy transcode of input JPEG file (decode to "
-            "pixels instead of doing lossless transcode).");
-
-DEFINE_bool(premultiply, false,  // TODO(tfish): Wire this up.
+DEFINE_bool(premultiply, false,
+            // TODO(tfish): Discuss - should `false` be treated differently from
+            // `unset` w.r.t. what default to use?
```

Comment:
I think we only have input formats that use non-premultiplied alpha, so I think it's a sensible default to have false here. If and when we would add an input format that (can) use premultiplied alpha, I think it would make sense to have a default that is "do the same as the input format", with an override to change it (which is a lossy operation in both directions).

Reply:
Ah right. So in that case I recommend having a default "do whatever the input format does".

Reply:
Turned into an int-flag to allow three options.